Thanks to Andres (?) for this page, from which these diffs come:
http://gnupoc.sourceforge.net/macosx.html

--- a/config.guess	Mon May 31 02:04:33 1999
+++ b/config.guess	Wed Mar 30 01:33:35 2005
@@ -225,6 +225,9 @@
     mvme88k:OpenBSD:*:*)
 	echo m88k-unknown-openbsd${UNAME_RELEASE}
 	exit 0 ;;
+    Power*Macintosh:Darwin:*:*)
+        echo powerpc-apple-darwin${UNAME_RELEASE}
+        exit 0 ;;
     powerpc:machten:*:*)
 	echo powerpc-apple-machten${UNAME_RELEASE}
 	exit 0 ;;
--- a/config.sub	Wed Aug  4 18:09:26 1999
+++ b/config.sub	Wed Mar 30 01:36:07 2005
@@ -945,6 +945,8 @@
 	      | -windows* | -osx | -abug |  -netware* | -os9* | -beos* \
 	      | -macos* | -mpw* | -magic* | -mon960* | -lnews* )
 		;;
+	-darwin*)
+		;;
 	-mac*)
 		os=`echo $os | sed -e 's|mac|macos|'`
 		;;
